rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -138,3 +138,93 @@
           }} // pubcloud slave
         }} // if
       }} // cit node
+
+- job:
+    name: JJB-artifacts-apt
+    node: ArtifactBuilder2
+    concurrent: false
+    build-discarder:
+      daysToKeep: 30
+    triggers:
+      - timed: "H 4 * * *"
+    parameters:
+      - pr-params
+      - string:
+          name: ghprbTargetBranch
```

Comment:
This is a parameter provided by the github pull request builder plugin, but the ghprb trigger is not enabled for this job, is that intentional?